addons/sale/views/sale_product_kanban_views.xml

```diff
@@ -0,0 +1,128 @@
+        <field name="name">sale.product.catalog.kanban.view</field>
+        <field name="model">product.product</field>
+        <field name="priority">999</field>
+        <field name="arch" type="xml">
+            <kanban
+                edit="0"
+                class="o_kanban_mobile"
+                action="set_sol_qty" type="object"
+                js_class="sale_product_kanban">
+                <field name="id"/>
+                <field name="name"/>
+                <field name="product_template_attribute_value_ids"/>
+                <field name="sol_qty"/>
+                <field name="currency_id"/>
+                <field name="default_code"/>
+                <templates>
+                    <t t-name="kanban-box">
+                        <div class="d-flex flex-column justify-content-between my-0 py-2 pe-4"
+                             t-attf-class="{{(record.sol_qty.value > 0) ? 'bg-primary' : ''}}">
+                            <div class="o_dropdown_kanban dropdown">
+                                <a role="button"
+                                   title="Dropdown menu"
+                                   class="dropdown-toggle o-no-caret btn"
+                                   data-bs-toggle="dropdown"
+                                   data-bs-display="static"
+                                   href="#">
+                                   <span class="fa fa-ellipsis-v"/>
+                                </a>
+                                <div class="dropdown-menu" role="menu">
+                                    <a role="menuitem"
+                                       class="dropdown-item"
+                                       type="object"
+                                       name="action_edit_template"
+                                       groups="sales_team.group_sale_manager">Edit</a>
+                                </div>
+                            </div>
+                            <div class="d-flex">
+                                <div class="o_kanban_image">
+                                    <img t-att-src="kanban_image('product.product', 'image_128', record.id.raw_value)"
+                                         class="o_image_64_contain"
+                                         alt="Product"/>
+                               </div>
+                                <div class="oe_kanban_details p-2 d-flex flex-column">
+                                    <div class="o_kanban_record_top flex-column w-100 justify-content-between">
+                                        <div class="o_kanban_record_title" t-attf-style="{{(record.sol_qty.value > 0) ? 'color: white' : ''}}">
+                                            <strong><field name="name"/></strong>
+                                        </div>
+                                        <div class="o_kanban_product_details" t-attf-style="{{(record.sol_qty.value > 0) ? 'color: white' : ''}}">
```

Comment:
The attribute `t-attf-style` is duplicated, I tried to set it as a value to avoid duplication but it does not work, it loses the reactivity.